README.md

```diff
@@ -11,45 +17,45 @@ Note that ruby 1.8 support was dropped in ruby-oci8 2.2.0.
 Use ruby-oci8 2.1.8 for ruby 1.8.
 
 What's new
-==========
+----------
 
 See {file:NEWS}.
 
 Sample one-liner
-================
+----------------
 
 Connect to scott/tiger, select `emp` and print as CSV format.
 
     ruby -r oci8 -e "OCI8.new('scott', 'tiger').exec('select * from emp') do |r| puts r.join(','); end"
 
 Homepage
-========
+--------
 
 * http://www.rubydoc.info/github/kubo/ruby-oci8
 
 Installation
-============
+------------
```

Comment:
Thanks for the suggestion @Gaurav2728! I had similar thoughts but ended up keeping it using `====`, `------` (it seemed just a bit more readable when looking at it in plain text).